tests/jvm/src/test/scala-2.12/scala/meta/tests/semanticdb/MemberSuite.scala

```diff
@@ -160,4 +160,42 @@ class MemberSuite extends DatabaseSuite(SemanticdbMode.Slim, MemberMode.All) {
     """.stripMargin
   )
 
+  members(
+    """
+      |object a {
+      |  val run = new java.lang.Runnable {
```

Comment:
Can we split these four cases into individual unit tests?

Reply:
Done